src/gax-internal/src/http.rs

```diff
@@ -356,4 +371,48 @@ mod test {
         assert_eq!(err.headers(), &Some(want));
         Ok(())
     }
+
+    #[tokio::test]
+    #[test_case(reqwest::StatusCode::OK, "{}"; "200 with empty object")]
+    #[test_case(reqwest::StatusCode::NO_CONTENT, "{}"; "204 with empty object")]
+    #[test_case(reqwest::StatusCode::NO_CONTENT, ""; "204 with empty content")]
+    async fn client_empty_content(code: reqwest::StatusCode, content: &str) -> TestResult {
+        let response = resp_from_code_content(code, content)?;
+        assert!(response.status().is_success());
+
+        let response = ReqwestClient::to_http_response::<wkt::Empty>(response).await;
+        assert!(response.is_ok());
+
+        let response = response.unwrap();
```

Comment:
nit: in general we prefer `?` over `unwrap()`. Not a big deal at all in tests, but we use `?` above, so why not use it here too.